packages/@aws-cdk/custom-resource-handlers/lib/custom-resources/aws-custom-resource-handler/construct-types.ts

```diff
@@ -19,4 +21,5 @@ export interface AwsSdkCall {
   readonly outputPath?: string;
   readonly outputPaths?: string[];
   readonly assumedRoleArn?: string;
-}
\ No newline at end of file
+  readonly logApiResponseData: boolean;
```

Comment:
Question on if this property should be optional or not. By the way we have set up the logic with `_render()`, we will always have a value for this field, so it can be "required". 

@colifran and I talked offline, I do not think that this is a problem. The main discussion is at what level we decide to set the default, and having it passed explicitly to the call is fine.

Reply:
For historical purposes, I want to make sure it is clear that the `AwsSdkCall` interface referenced above is not a type used in the `aws-cdk-lib`. Instead, this is a mirror of `AwsSdkCall` which is defined as part of the `AwsCustomResource` construct. However, the format of this represents the `AwsSdkCall` type as it would be received by the v2 and v3 handler, e.g., after `render` has been called on `Logging` and all `logging` properties are rendered. This interface just exists to provide a strongly typed `call` in the v2 and v3 handlers.

My opinion is that we should set the default prior to passing the event so that the event properties are visible in the synthesized CloudFormation template.